lib/template-mixins.js

```diff
@@ -247,6 +249,7 @@ module.exports = function (fields, options) {
                     return {
                         label: t(label) || '',
                         value: value,
+                        type: opts.type,
                         selected: selected,
```

Comment:
Does 'selected' need some extra handling to cater for an array of selections? For example https://github.com/UKHomeOffice/passports-template-mixins/blob/test-feature/multiple-checkboxes/lib/template-mixins.js#L245, so that multiple selections show as ticked. Maybe selected values should also be unit tested.